docs/docs/configuration/filtering.md

```diff
@@ -494,4 +494,3 @@ Please refer to the [VEX documentation](../supply-chain/vex.md) for the details.
 
 
 [^1]: license name is used as id for `.trivyignore.yaml` files.
-[^2]: This doesn't work for package licenses. The `path` field can only be used for license files (licenses obtained using the [--license-full flag](../scanner/license.md#full-scanning)).
```

Comment:
Also, as i wrote bellow, OS packages (apk, dpkg, rpm) don't fill FilePath field. So we need write about this.

Reply:
I fixed the info about os packages.

As for `go` I don't believe this functionality works since in json license report `FilePath` for go modules is empty. I tried to debug that quickly and this confirms.

When `filterLicenses` gets executed at lines:
```
// Filter by ignore file
if f := ignoreConfig.MatchLicense(l.Name, l.FilePath); f != nil {
```
...although `ignoreConfig` contains license entries as per prepared `trivyignore.yaml`, `l.FilePath` is empty so no filtering is applied.

Reply:
I found the problem and suggested a fix for this problem - https://github.com/aquasecurity/trivy/pull/6215#discussion_r1506989848

Reply:
After we passed logic for lockfile -  i think we need to add 2 tests [here](https://github.com/aquasecurity/trivy/blob/6ccc0a554b07b05fd049f882a1825a0e1e0aabe1/pkg/scanner/local/scan_test.go):
- for application scan (e.g. Python METADATA)
- for lockfile (e.g. go.mod)

Can you do this?

Reply:
Added one test with two applications

Reply:
Great! Thanks!